### ISSUE TITLE
I want to not mapping name form value. when getValues() or onSubmit.

### ISSUE THREAD

**Issue by youngje4284** [react-hook-form/react-hook-form#1291]
Please consider asking the question at our spectrum channel.

https://spectrum.chat/react-hook-form

**Describe the question?**

I get data from server and i mapping use reset.
my project server data include 'fromServerId' key, this key required for array data fix.
so i want to included fromServerId key value when onSubmit event. 
but getValues() and onSubmit values not paramter not include fromServerId key and value.
i think fromServerId not mapping name in form.

in short, how not mapping name data get to use getValues() or onSubmit() values parameter?  

**To Reproduce**

**Codesandbox link (Required)**
https://codesandbox.io/s/react-hook-form-usefieldarray-ccp7s

**Screenshots**

**Additional context**

**Comment by bluebill1049** [comment]
use hidden input.

**Comment by youngje4284** [comment]
ok.  then if you have plan make to this requirement in future?

**Comment by bluebill1049** [comment]
no, i don't think so. it's a HTML form standard that when you submit a form should only contains what's in the inputs. I understand controlled input allow you to do so without an actual input, but with RHF i prefer to stay closer with HTML standard for form.
